app/components/EventsList.tsx

```diff
@@ -5,19 +5,25 @@ import * as styles from "./EventsList.css";
 import { Heading } from "./Heading";
 
 export interface EventsListProps {
+	active?: boolean;
+	descriptor: string;
```

Comment:
```suggestion
	descriptor: "All" | "Upcoming";
```
Thoughts? Unnecessarily restrictive?

Reply:
I'm happy either way! Let's try it out. 🚀